tests/Rules/UnusedPublicClassMethodRule/templates/nested/nested_template_used_method.twig

```diff
@@ -0,0 +1 @@
+{{ some_value.anotherValue }}
```

Comment:
new test-case to cover the nested reading functionality

Reply:
Perfect :clap: that's very useful :+1: